src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/api/summaries/temperature/ClusterDimensionalSummary.java

```diff
@@ -70,13 +71,19 @@
     /**
      * This is the mean temperature for this dimension over all the nodes in the cluster.
      */
-    private TemperatureVector.NormalizedValue meanTemperature;
+    private TemperatureVector.VectorValues meanTemperature;
+
+    /**
+     * This is the average value used over shards.
+     */
+
+    private double avgMetricValueUsedOverNodes;
```

Comment:
The `VectorValues` class now contains the raw value and the normalized value. So, shouldn't this be already covered in the rawValue part ? Do we really need a different variable for this ?